src/xb-silo.c

```diff
@@ -824,9 +819,6 @@ xb_silo_load_from_file (XbSilo *self,
 
 	/* no longer valid */
 	g_hash_table_remove_all (priv->file_monitors);
-	g_hash_table_remove_all (priv->nodes);
-	g_hash_table_remove_all (priv->strtab_tags);
-	g_clear_pointer (&priv->guid, g_free);
```

Comment:
this looks wrong

Reply:
Oops, I meant to drop that. It’s a change I was playing around with earlier: that code was redundant because `xb_silo_load_from_bytes()` (which is called later in this control flow) does the same thing. But I think it’s clearer to leave them in place.